tests/kernel/fatal/src/main.c

```diff
@@ -4,6 +4,13 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+/**
+ * @defgroup kernel_fatal_tests Fatal
```

Comment:
it it not justified to have a group for each test, put this under common. (kernel_common_tests)

Reply:
@nashif Added under kernel_common_tests.